WordPressKit/PostServiceRemoteREST.h

```diff
@@ -53,4 +55,18 @@
                    success:(void (^)(RemotePost *))success
                    failure:(void (^)(NSError *error))failure;
 
+/**
+ *  @brief      Requests a list of users that liked the post with the specified ID.
+ *
+ *  @discussion Due to the API limitation, up to 90 users will be returned from the
+ *              endpoint.
+ *
+ *  @param      postID      The ID for the post. Cannot be nil.
+ *  @param      success     The block that will be executed on success. Can be nil.
+ *  @param      failure     The block that will be executed on failure. Can be nil.
+ */
+- (void)getLikesForID:(NSNumber *)postID
```

Comment:
Can we make the method name a bit clearer? `ID` is a bit vague. Maybe something like `getLikesforPost` instead?

Reply:
Yeah, sure! Would `getLikesForPostID` be better here? Because there are other methods that require a `RemotePost` type as the first parameter, such as `-getAutoSaveForPost:(RemotePost *)post`.

Reply:
Yep, that'll work too!